### PR TITLE
chore: update download link

### PR DESCRIPTION
**Pull request by ybw0014**
## Description

Update the download link in README.md to blob builds.

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [ ] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.16.* - 1.20.*).
- [ ] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Comment by JustAHuman-xD** [comment]
problems:
1. Those are direct downloads compared to the previous page, idk if thats a good idea
2. there are also download things on the wiki (which can be addressed in a separate pr) (https://github.com/Slimefun/Wiki/blob/master/pages/Installing-Slimefun.md)

**Comment by JustAHuman-xD** [comment]
> problems:
> 
> 1. Those are direct downloads compared to the previous page, idk if thats a good idea
> 2. there are also download things on the wiki (which can be addressed in a separate pr) (https://github.com/Slimefun/Wiki/blob/master/pages/Installing-Slimefun.md)

The first problem is what allowed you to snipe this PR from me lol, I was trying to find a way to not do direct downloads

**Comment by ybw0014** [comment]
the release channels are displayed together, unless walshy adds something to separately show them

**Comment by WalshyDev** [comment]
https://github.com/WalshyDev/blob-builds/pull/15

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Here is a full summary of the differences between the two different versions of
 | **Bug Reports** | :heavy_check_mark: | :x: |
 | **testing before release** | :x: | :heavy_check_mark: |
 | **change logs** | :x: | :memo: **[change log](https://github.com/Slimefun/Slimefun4/blob/master/CHANGELOG.md)** |
-| **Download link** | :floppy_disk: **[download latest](https://thebusybiscuit.github.io/builds/TheBusyBiscuit/Slimefun4/master/)** | :floppy_disk: **[download "stable"](https://thebusybiscuit.github.io/builds/TheBusyBiscuit/Slimefun4/stable/)** |
+| **Download link** | :floppy_disk: **[download latest](https://blob.build/dl/Slimefun4/Dev/latest)** | :floppy_disk: **[download "stable"](https://blob.build/dl/Slimefun4/RC/latest)** |
```
Comment (WalshyDev) [inline README.md:42]:
```suggestion
| **Download link** | :floppy_disk: **[download latest](https://blob.build/project/Slimefun4/Dev)** | :floppy_disk: **[download "stable"](https://blob.build/project/Slimefun4/RC)** |
```

**Review by WalshyDev — DISMISSED** [review]

**Review by J3fftw1 — APPROVED** [review]

**Review by Sfiguz7 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Here is a full summary of the differences between the two different versions of
 | **Bug Reports** | :heavy_check_mark: | :x: |
 | **testing before release** | :x: | :heavy_check_mark: |
 | **change logs** | :x: | :memo: **[change log](https://github.com/Slimefun/Slimefun4/blob/master/CHANGELOG.md)** |
-| **Download link** | :floppy_disk: **[download latest](https://thebusybiscuit.github.io/builds/TheBusyBiscuit/Slimefun4/master/)** | :floppy_disk: **[download "stable"](https://thebusybiscuit.github.io/builds/TheBusyBiscuit/Slimefun4/stable/)** |
+| **Download link** | :floppy_disk: **[download latest](https://blob.build/project/Slimefun4/Dev)** | :floppy_disk: **[download "stable"](https://blob.build/project/Slimefun4/RC)** |
```
Comment (J3fftw1) [inline README.md:42]:
```suggestion
| **Download links** | :floppy_disk: **[download latest](https://blob.build/project/Slimefun4/Dev)** | :floppy_disk: **[download "stable"](https://blob.build/project/Slimefun4/RC)** |
```

**Review by J3fftw1 — APPROVED** [review]

**Pull request closed without merging**